backup_restore_comparison.py

```diff
@@ -0,0 +1,338 @@
+import json
+import os
+import re
+import threading
+import uuid
+from enum import Enum
+
+from argus.client.generic_result import Status
+from mgmt_cli_test import ManagerTestFunctionsMixIn
+from sdcm import mgmt
+from sdcm.argus_results import ManagerBackupReadResult, ManagerBackupBenchmarkResult, submit_results_to_argus
+from sdcm.cluster import BaseNode
+from sdcm.mgmt import TaskStatus
+from sdcm.remote import shell_script_cmd
+from sdcm.sct_events.system import InfoEvent
+from sdcm.utils.node import build_node_api_command, RequestMethods
+from sdcm.utils.time_utils import ExecutionTimer
+
+
+class ManagerReportType(Enum):
+    READ = 1
+    BACKUP = 2
```

Comment:
Read/Backup values do not make sense to me? Is it meant to be Restore time/Backup time?

Reply:
changed to read_write and backup_restore